data/Act.csv

```diff
@@ -6,136 +6,82 @@ A00004,P0108,AT01,T00010,,1972,1975,Xu Xiao read Ticket to the Stars and was ver
 A00005,P0014,AT05,T00010,P0108,1972,1975,Zhao Yifan copied Ticket to the Stars and handed it to Xu Xiao,T00012,381,,2017-07-12,OS,2017-07-12,OS
 A00006,P0109,AT04,T00013,,1969,1969,Shu Ting wants to go to Gorky‘s 'Universities' during her rustication,T00014,300,,2017-07-19,LH,2017-10-02,DP
 A00007,P0020,AT04,T00013,,1966,1976,Gao Hua mentions that during the turmoil years of CR; when as a son of 'heiwulei' (five black categories) he has been living in the bottom of the society;  he received supercilious looks and was discriminated against. During these hard times these two books by Gorky acted as his spiritual guides.,T00006,133,,2017-07-19,LH,2017-10-02,DP
-A00008,P0009,AT01,T00022,,1966,,They started reading neibu books before rustication time already. Kafka's 'Trial' was among the most influential neibu books for Bei Dao,T00011,80,,2017-07-19,OS,2017-10-02,DP
-A00009,P0014,AT03,T00024,P0108,1966,1970?,Zhao Yifan gave the book as a shouchaoben to Xu Xiao,T00012,381,,2017-07-25,MG,2017-07-25,MG
+A00008,P0009,AT01,T00022,,1966,19XX,They started reading neibu books before rustication time already. Kafka's 'Trial' was among the most influential neibu books for Bei Dao,T00011,80,,2017-07-19,OS,2018-01-22,MG
+A00009,P0014,AT03,T00024,P0108,1966,19XX,Zhao Yifan gave the book as a shouchaoben to Xu Xiao,T00012,381,,2017-07-25,MG,2018-01-22,MG
```

Comment:
Wouldn't "1970?" be more precise, as it shows that it happened somewhere around that date?